core/src/main/java/org/web3j/protocol/websocket/events/Log.java

```diff
@@ -0,0 +1,46 @@
+package org.web3j.protocol.websocket.events;
+
+import java.util.List;
+
+public class Log {
```

Comment:
I presume this Log differs from the other Log objects already defined?

Reply:
Yes, existing `Log` class contains the following fields:

```java
    private boolean removed;
    private String logIndex;
    private String transactionIndex;
    private String transactionHash;
    private String blockHash;
    private String blockNumber;
    private String address;
    private String data;
    private String type;
    private List<String> topics;
```

while this `Log` class contains the following fields:

```java
    private String address;
    private String blockHash;
    private String blockNumber;
    private String data;
    private String logIndex;
    private List<String> topics;
    private String transactionHash;
    private String transactionIndex;
```